code/__DEFINES/mobs.dm

```diff
@@ -166,6 +166,13 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define XENO_SLASHING_ALLOWED 1
 #define XENO_SLASHING_RESTRICTED 2
 
+// =============================
+// xeno building
+#define XENO_BUILDING_FORBIDDEN (1<<0)
```

Comment:
this is useless, it's forbidden if neither building silo and building turret is off

Reply:
Done